### PR TITLE
lump:dehacked.lmp

### PR DESCRIPTION
**Pull request by Voros2**
Added better story screens for Phase 1.

Minor edits to Phase 2's story screens.

All levels in Phase 2 have been named:
-MAP06 (Voltcom9)
-MAP09 (Kristien "sajbear" Nilsen)
-MAP10 (Jewellds)
-MAP26 (DragonMorpheus)
-MAP27 (Voltcom9)

All levels, but C3M5, in Phase 1 have been named:
-C2M3 (Voltcom9)
-C3M4 (Voltcom9)

All other changes were made by me.

### CODE REVIEW

File: lumps/dehacked.lmp [hunk 1]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
-         carnage!\n\n\
-         -or play Chapter 4: Double Impact!
-E4TEXT = Despite having an entire army against\n\
-         you, you manage to reach the\n\
+E1TEXT = This Site is safe now.\n\
+        Sure, everyone's dead, but so are they.\n\
+        The monsters that infested this place\n\
+        no longer exist in this reality. But it\n\
+        feels wrong. Too simple. They never stopped\n\
+        you only slowed you down. Why?\n\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: add comma: never stopped you, only slowed you down

File: lumps/dehacked.lmp [hunk 2]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
+        What were they hiding from you so\n\
+        desperately anyway?\n\n\        
+        You step on the teleporter hoping to find\n\
+        out right now.
+E2TEXT = The brutes rip themselves apart\n\
+        as they fall on the soft ground\n\
+        blood leaking.\n\
+        metal burning.\n\
+        They're dead. But is it over? No.\n\n\
+        The soil beneath your feet start to quake.\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: *starts to

File: lumps/dehacked.lmp [hunk 3]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
-         out of your mind.\n\n\
-         Guess you'll find out soon.
+         outer storage warehouse. You hear growls.\n\
+         A squadron was sent here, judging from the\n\
+         two 4x4 vehicles parked near the gates.\n\n\
+         So where are they?\n\n\
+         Most likely dead. Or turned into monsters.\n\
+         Best to end their misery now than later.\n\
+         You loot the vehicles and open the gates.\n\
+         The lift hums as you enter, rising down.\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: Rising down?

File: lumps/dehacked.lmp [hunk 4]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
-C2TEXT = You make your way to the outskirts of\n\
-         the city. This was once the planet's\n\
-         most important economic powerhouse,\n\
-         and headquarters of the AGM conglomerate.\n\n\
-         Now it's a wreck.\n\n\
-         Every entrance has been heavily fortified,\n\
-         but you find one small forgotten doorway,\n\
-         it must be an industrial loading bay\n\
-         of some kind. Guess this is your way in.
+C2TEXT = You stand on ledge, looking down\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: *On the ledge

File: lumps/dehacked.lmp [hunk 5]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
-         any more.\n\n\
-         The saying goes, live free or die.\n\
-         Guess that doesn't always mean the\n\
-         latter.
+C4TEXT = The gigantic beast becomes unstable.\n\
+        Its final roar echoes throughout the room\n\
+        till it crumples in a billow of smoke and a\n\
+        pool of blood and molten metal.\n\n\
+        A portal opens up, and in it, you see a small\n\
+        town. You went in, forgetting the past.\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: went -> go

File: lumps/dehacked.lmp [hunk 6]
```diff
@@ -318,88 +318,117 @@ NIGHTMARE = this is the highest skill level.\n\
...
+        No one will know know who saved them.\n\n\
+        No one will know what happened here.\n\n\
+        No one will ever find you again.
 # Before MAP31 (secret level #1):
-C5TEXT = You step onto the teleport and after a\n\
+C5TEXT = You step onto the machine and after a\n\
          strange flash you find yourself in an\n\
          old castle. What is this place? Before\n\
-         you can wonder further, you hear\n\
+         you can wander further, you hear\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: This wasn't a spelling mistake, though the change works anyway :)

**Comment by fragglet** [comment]
Mostly looks good! If you can just address the few comments then I'll happily merge.

**Comment by CraftedPixel** [comment]
Nice job!

**Comment by Voros2** [comment]
Done.

File: lumps/dehacked.lmp [hunk 7]
```diff
@@ -322,8 +322,8 @@ E1TEXT = This Site is safe now.\n\
         Sure, everyone's dead, but so are they.\n\
         The monsters that infested this place\n\
         no longer exist in this reality. But it\n\
-        feels wrong. Too simple. They never stopped\n\
-        you only slowed you down. Why?\n\n\
+        feels wrong. Too simple. They never stopped,\n\
```
Comment (fragglet) [inline lumps/dehacked.lmp]: You took out the word "you" and this doesn't necessarily make sense now. They didn't stop what?

The correct text should probably be: They never stopped you, only slowed you down.

File: lumps/dehacked.lmp [hunk 8]
```diff
@@ -173,10 +173,10 @@ CC_DEMON = flesh worm
 CC_LOST = deadflare
 CC_CACO = orb monster
 CC_HELL = pain bringer
-CC_BARON = pain lord
+CC_BARON = pestmeister
```
Comment (fragglet) [inline lumps/dehacked.lmp]: Not sure about "pestmeister" as a monster name. "Pest" usually implies something small, verminous and irritating, like a fly or a mouse. It's also nice to have some kind of commonality of name between the Hell Knight / Baron of Hell names.

**Comment by Voros2** [comment]
Raymoohawk's future Pain Bringer replacement is what he calls the Pestmeister. 

I think the "pest" is the player, and its job is to get rid of "pests".

And yeah, I'll add the "you" back.

**Comment by Voros2** [comment]
If you don't mind, I'd also like to update the PAR times. So if you do wish to merge this PR, can you do so after I update the PAR times? I've already begun recording my times. I don't do speedruns, so they'll be legit times. I'll also add an extra 30-60 seconds, depending on how good or how bad I played on the level, on every PAR I get. 
:)

**Comment by Voros2** [comment]
Ok maybe not at the moment, seeing how some of the maps are extremely confusing.

I'll do them next time.

**Comment by fragglet** [comment]
Let's go with pestmeister for now - we can always change it again later if people really don't like it. New par times sound nice but it's better to get this merged now and update those later.

**Comment by fragglet** [comment]
Oh, you changed pestmeister back anyway - never mind then :) Anyway, merged now.

**Pull request merged**